entries/a/acorns.com.json

```diff
@@ -3,14 +3,16 @@
     "domain": "acorns.com",
     "url": "https://www.acorns.com/",
```

Comment:
```suggestion
```

Reply:
Good catch, I will start paying attention to this more as I go through the sites at this time.